src/Frontend/Core.hs

```diff
@@ -676,7 +684,9 @@ coreRunHandler mp mr = do
                 AccessDenied s u -> handleDenied s u
                 AccessDeferred -> throwError500 "AccessDeferred should not be used with NotLoggedIn"
   where
-    handleDenied s u = throwServantErr $ (maybe Servant.err403 err303With u) { errBody = cs s }
+    handleDenied s u = do
+        logEvent WARN $ cs s
+        throwServantErr $ (maybe Servant.err403 err303With u) { errBody = cs s }
         -- FIXME log these events as INFO, should we do this here or more globally for servant errors.
```

Comment:
You should generally look out more for FXMEs near your code and update/remove them if appropriate.